src/main/java/com/github/manolo8/darkbot/utils/FileUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.github.manolo8.darkbot.utils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileUtils {
+    public static void createDirectory(Path path) {
```

Comment:
Probably shound name `ensureDirectoryExists` to indicate it may not always create it